cpp/tests/datetime/datetime_ops_test.cpp

```diff
@@ -104,96 +94,6 @@ TEST_F(BasicDatetimeOpsTest, TestExtractingDatetimeComponents)
       987234623   // 1970-01-01 00:00:00.987234623 GMT
     };
 
-  CUDF_TEST_EXPECT_COLUMNS_EQUAL(*extract_year(timestamps_D),
```

Comment:
These should be changed to call the `extract_datetime_component` and not just removed.

Reply:
I think this is already happening below: the new usage is already tested: https://github.com/rapidsai/cudf/blob/7ea8f32877ec59197870d39a54c40bd9d744dade/cpp/tests/datetime/datetime_ops_test.cpp#L97

Reply:
Yep, this test uses `extract_datetime_component` already. But I added the others ones I deleted back